dev/com.ibm.ws.ras.instrument_test/test/com/ibm/example/bytecode/HelloWorldJava11.java

```diff
@@ -0,0 +1,21 @@
+package com.ibm.example.bytecode;
```

Comment:
How can these be in the `test` directory and not be built?  All classes in the test directory are going to be built as unit tests.  Also all of these java files need to have the copyright added to them.

Reply:
I updated the README to me more accurate.  They are compiled, but the tests doesn't use the automatically compiled classes, it will use the cached class files.